Neos.Flow/Classes/Composer/InstallerScripts.php

```diff
@@ -71,7 +71,7 @@ public static function postPackageUpdateAndInstall(PackageEvent $event)
         if (!$operation instanceof InstallOperation && !$operation instanceof UpdateOperation) {
             throw new Exception\UnexpectedOperationException('Handling of operation with type "' . $operation->getJobType() . '" not supported', 1348750840);
```

Comment:
Looking at that again I wonder… what happens if that line is executed and `getJobType()` is called?

Reply:
good call.. this will probably break